Bio/SeqIO/PhdIO.py

```diff
@@ -55,11 +55,13 @@
 
 from Bio.SeqRecord import SeqRecord
 from Bio.Sequencing import Phd
-from Bio.SeqIO import QualityIO
+from .QualityIO import _get_phred_quality
 from .Interfaces import SequenceWriter
 
+from typing import Any, Generator, IO, TextIO, Union
 
-def PhdIterator(source):
+
+def PhdIterator(source: Union[TextIO, str]) -> Generator[SeqRecord, None, None]:
```

Comment:
General query, having read https://docs.python.org/3/library/typing.html#typing.Generator but not tried this:

Why use ``Generator[SeqRecord, None, None]`` over the seemingly clearer ``Iterator[SeqRecord]`` for the return?

Reply:
Good catch, I'll fix this